girder_worker/__init__.py

```diff
@@ -6,3 +7,24 @@
 _cfgs = ('worker.dist.cfg', 'worker.local.cfg')
 config = SafeConfigParser(os.environ)
 config.read([os.path.join(PACKAGE_DIR, f) for f in _cfgs])
+
+
+class GirderWorkerPluginABC(object):
```

Comment:
I think this class and/or `GirderWorkerPlugin` could use a bit of explanation in the docs. @kotfic has shown me how this works and it is cool, extensible, easy, etc. but the RTD could perhaps use a simple inlined code example of how to make a GW plugin.